src/query/cellxgene_census/test.py

```diff
@@ -0,0 +1,32 @@
+import sys
+import os
+import pytest
+import unittest
+
+## VIASH START
+meta = {
+    'resources_dir': './resources_test/'
+}
+## VIASH END
+
+OUTPUT_FILE = "output.h5mu"
+
+
+def test_cellxgene(run_component):
+    run_component([
+        "--input_database", "CellxGene",
+        "--modality", "rna",
+        "--cell_ontology_release", "2023-05-22",
+        "--cellxgene_release", "2023-05-15",
+        "--species", "homo_sapiens",
+        "--tissue", "lung",
+        "--obs_column_names", "disease",
+        "--output", OUTPUT_FILE,
+        "--metadata_only", "True"
+    ])
+
+    # check whether file exists
+    assert os.path.exists(OUTPUT_FILE), "Output file does not exist"
+
+if __name__ == '__main__':
+    sys.exit(pytest.main([__file__, "--capture=no"], plugins=["viashpy"]))
```

Comment:
The plugins used to be required because of a pytest issue, but this now works without.
I would also like to not use `"--capture=no"` because this keeps us from seeing the component output in the CI. `viashpy` outputting way to much info was also solved in version `0.3.0`: https://github.com/viash-io/viashpy/blob/main/CHANGELOG.rst#030-06062023
```suggestion
if __name__ == '__main__':
    sys.exit(pytest.main([__file__]))
```